apps/vdccreate/vdccreate.cpp

```diff
@@ -162,7 +186,15 @@ void	set_coord(
 	if (rc<0) exit(1);
 }
 
-void	set_coords(
+void	set_coord_stretched(
+	VDCNetCDF	&vdc,
+	string dimname, const vector <float> &coords
+) {
+	int rc = vdc.PutVar(0, dimname, -1, coords.data());
```

Comment:
I see that VDC::PutVar has two forms, one that applies to a defined timestep (0 in this case) and another that applies to all timesteps in the VDC.  Is there's a rule that coordinate variables only need to be written to timestep 0?

Reply:
If the coordinate variables are static (i.e. don't vary with time) they do not have a time index. vdccreate does not support time-varying coordinate variables.